src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -745,7 +745,7 @@ def load_arguments(self, _):
             c.argument('publisher', help='The name of the extension publisher.')
             c.argument('settings', type=validate_file_or_dict, help='Extension settings in JSON format. A JSON file path is also accepted.')
             c.argument('protected_settings', type=validate_file_or_dict, help='Protected settings in JSON format for sensitive information like credentials. A JSON file path is also accepted.')
-            c.argument('version', help='The version of the extension')
+            c.argument('version', help='The version of the extension. If you want to use the exact version number specified, please specify --no-auto-upgrade.')
```

Comment:
To pin extension version to fix version, pls specify --no-auto-upgrade.
```suggestion
            c.argument('version', help='The version of the extension. To pin extension version to specific version number, please specify --no-auto-upgrade.')
```